tests/unit/sqlalchemy/test_compiler.py

```diff
@@ -29,6 +30,13 @@
     Column('id', Integer, primary_key=True),
     Column('name', String),
 )
+table_with_catalog = Table(
+    'table',
+    metadata,
+    Column('id', Integer, primary_key=True),
```

Comment:
Is the `primary_key=True` required for the test? I don't think so. If so please remove it since it distracts attention from what the test and this change is actually about.

Reply:
Same for the DDL which is used below - the `PRIMARY KEY` seems unrelated to the feature being added.

Reply:
I see that the `primary_key=True` just above this one is pre-existing but that should also be removed. Unfortunate that it didn't get caught when it was introduced.

Reply:
I went ahead and removed this from both temp tables, as well as in this test.